conda/gateways/repodata/__init__.py

```diff
@@ -48,10 +53,16 @@
 stderrlog = logging.getLogger("conda.stderrlog")
 
 
-# if repodata.json.zst or repodata.jlap were unavailable, check again after this
-# amonut of time.
+# if repodata.json.zst or repodata.jlap were unavailable, check again later.
 CHECK_ALTERNATE_FORMAT_INTERVAL = datetime.timedelta(days=7)
 
+# repodata.info/state.json keys to keep up with the CEP
+LAST_MODIFIED_KEY = "last_modified"
+ETAG_KEY = "etag"
+CACHE_CONTROL_KEY = "cache_control"
+URL_KEY = "url"
+CACHE_STATE_EXTENSION = ".info.json"
```

Comment:
XXX figure out how to fetch the current CEP, pull out the relevant field names, and use them here :-P